tools/cockpit.spec

```diff
@@ -358,6 +358,16 @@ Cockpit support for remoting to other servers, bastion hosts, and a basic dashbo
 # HACK: Until policy changes make it downstream
 # https://bugzilla.redhat.com/show_bug.cgi?id=1381331
 test -f %{_bindir}/chcon && chcon -t cockpit_ws_exec_t %{_libexecdir}/cockpit-ssh
+%if 0%{?fedora} > 0 && 0%{?fedora} >= 26
+if type semodule >/dev/null 2>&1; then
```

Comment:
echo "HACK: Workaround for broken SELinux policy: https://bugzilla.redhat.com/show_bug.cgi?id=1381331" > &2